common/api/core-frontend.api.md

```diff
@@ -7623,6 +7625,15 @@ export namespace PerModelCategoryVisibility {
         clearOverrides(modelIds?: Id64Arg): void;
         getOverride(modelId: Id64String, categoryId: Id64String): Override;
         setOverride(modelIds: Id64Arg, categoryIds: Id64Arg, override: Override): void;
+        setOverrides(perModelCategoryVisibility: Props[], iModel?: IModelConnection): Promise<void>;
+    }
+    export interface Props {

```

Comment:
This is implicitly `@public` - is that intentional?

Reply:
What would you suggest? Beta feels safest I suppose, but DR and OCP would likely use this. Am I allowed to have an interface within a public namespace that isn't also public?

Reply:
I just wanted to make sure it was intended to be public, and if so, that it is properly documented.
Yes, each member of a namespace can have a "lower" release tag than the namespace itself. If you think it might need to change in a backward-incompatible way, beta's fine.

Reply:
Please document each member of the interface.

Reply:
Changed it, just not sure if the comments are sufficient though.